### PR TITLE
Fixed Django guide mistakenly mentioning FastAPI

### PR DESCRIPTION
**Pull request by sergeykolosov**
Recent changes in Django guide mention FastAPI where it meant to be Django.

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Review by lizokm — APPROVED** [review]
Thanks for the great catch, Sergey!

**Comment by sergeykolosov** [comment]
Well, apparently I can't do anything about this PR but wait for maintainers to merge it. @lizokm shall I just ignore bot messages/emails?

**Comment by lizokm** [comment]
It looks like the change has been made so I'll go ahead and close this out. Thanks again!

**Pull request closed without merging**
